ios/Classes/PTFlutterDocumentController.m

```diff
@@ -1009,7 +1009,25 @@ - (void)applyViewerSettings
     
     // Annotation Manager
     if (self.isAnnotationManagerEnabled && self.userId) {
-        PTExternalAnnotManager* annotManager = [self.pdfViewCtrl EnableAnnotationManager:self.userId mode:e_ptadmin_undo_others];
+        // Edit Mode
+        if ([PTAnnotationManagerEditModeOwnKey isEqualToString:self.annotationManagerEditMode]) {
+            self.toolManager.annotationManager.annotationEditMode = PTAnnotationModeEditOwn;
+        } else if ([PTAnnotationManagerEditModeAllKey isEqualToString:self.annotationManagerEditMode]) {
+            self.toolManager.annotationManager.annotationEditMode = PTAnnotationModeEditAll;
+        }
+        
+        self.toolManager.annotationAuthor = self.userId;
+        self.toolManager.annotationManager.annotationAuthorIdentifier = self.userId;
+        self.toolManager.annotationPermissionCheckEnabled = YES;
+        self.toolManager.annotationAuthorCheckEnabled = YES;
```

Comment:
I'm not sure we should have author check enabled - @dluco ?

Reply:
The annotation edit modes require the `annotationAuthorCheck` to be enabled in order to work - it's possible that setting the `annotationEditMode` should automatically enable the author check (in the SDK).